tests/plugins/preview/manual/previewcdn.md

```diff
@@ -0,0 +1,16 @@
+@bender-tags: 4.16.2, 4444, bug
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, print, font, colorbutton, format, clipboard, pagebreak, toolbar, floatingspace, link, image2
+
+## Steps
+1. Wait for the editor to load.
+1. Click preview button.
+1. Open browser console in the new tab.
+
+## Expected result
+New browser tab with preview are opened correctly with no error in browser console.
```

Comment:
> New browser tab with preview are opened correctly

This should be expected result after 2nd step - you click preview and new tab/window with correct content is shown.

> with no error in browser console

And this is expected result after 3rd step.

Reply:
I'll correct it. Thanks.